sync-core/src/main/java/com/cloudant/sync/query/QueryValidator.java

```diff
@@ -26,10 +26,19 @@
  */
 class QueryValidator {
 
-    public static final String AND = "$and";
-    public static final String OR = "$or";
-    public static final String EQ = "$eq";
-
+    private static final String AND = "$and";
```

Comment:
This is something thats been on my radar as well.  Its done in 24a70d3e1a88cbd2183cca6f5dea90cc598a0d5f.